src/internal/api/packages/sbom.go

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package packages provides api functions for managing Zarf packages.
+package packages
+
+import (
+	"net/http"
+	"net/url"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"syscall"
+
+	"github.com/defenseunicorns/zarf/src/config"
+	"github.com/defenseunicorns/zarf/src/internal/api/common"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	"github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/defenseunicorns/zarf/src/types"
+	"github.com/go-chi/chi/v5"
+	"github.com/mholt/archiver/v3"
+)
+
+var signalChan = make(chan os.Signal, 1)
+
+// ExtractSBOM Extracts the SBOM from the package and returns the path to the SBOM
+func ExtractSBOM(w http.ResponseWriter, r *http.Request) {
+	path := chi.URLParam(r, "path")
+
+	sbom, err := extractSBOM(path)
+
+	if err != nil {
+		message.ErrorWebf(err, w, err.Error())
+	} else {
+		common.WriteJSONResponse(w, sbom, http.StatusOK)
+	}
+
+}
+
+// DeleteSBOM removes the SBOM directory
+func DeleteSBOM(w http.ResponseWriter, _ *http.Request) {
+	err := cleanupSBOM()
+	if err != nil {
+		message.ErrorWebf(err, w, err.Error())
+		return
+	}
+	common.WriteJSONResponse(w, nil, http.StatusOK)
+}
+
+// cleanupSBOM removes the SBOM directory
+func cleanupSBOM() error {
+	err := os.RemoveAll(config.ZarfSBOMDir)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// Extracts the SBOM from the package and returns the path to the SBOM
+func extractSBOM(escapedPath string) (sbom types.APIPackageSBOM, err error) {
+	const sbomDir = "zarf-sbom"
+	const SBOM = "sboms.tar"
```

Comment:
```suggestion
```

`zarf-sbom` is not used and `SBOM` could be placed next to `config.ZarfSBOMDir` as `config.ZarfSBOMTar` replacing the other places the string literal `sboms.tar` is used (`src/pkg/packager/inspect.go`, `src/pkg/packager/common.go`, and `src/cmd/tools/archiver.go`)